.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,8 +14,9 @@ This will help reviewers and should be a good start for the documentation.
 Additionally (see https://symfony.com/releases):
  - Always add tests and ensure they pass.
  - Bug fixes must be submitted against the lowest maintained branch where they apply
-   (lowest branches are regularly merged to upper ones so they get the fixes too.)
+   (lowest branches are regularly merged to upper ones so they get the fixes too).
  - Features and deprecations must be submitted against the latest branch.
+ - Describe, if possible, a code usage for documentation purposes.  
```

Comment:
```suggestion
 - For new features, provide some code snippets to help understand usage.  
```

Reply:
I think keeping the word `documentation` is worth it, 
what about:

```suggestion
- For new features, provide some code documentation to help understand usage. 
```

Reply:
That does not mean the same for me. "code documentation" sounds like documenting the code of the PR.

Reply:
the word documentation is on L12 :)

Reply:
oki lets try this for now thx!